site/static/notebooks/Autograph_DEMO.ipynb

```
@@ -0,0 +1,966 @@
+    "        method,\n",
+    "        f\"{SERVER_URL}{suffix}\",\n",
+    "        json=payload,\n",
+    "        headers={\n",
+    "            \"Authorization\": f\"Bearer {jwt_token}\",\n",
+    "            \"Content-Type\": \"application/json\",\n",
+    "        },\n",
+    "        verify=VERIFY_TLS,\n",
+    "    )\n",
+    "    response.raise_for_status()\n",
+    "    return response.json() if response.content else {}\n",
+    "\n",
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)